src/shared/form_fields/member.js

```diff
@@ -149,12 +142,13 @@ const validate = (values) => {
   const add_test = (tests, key) =>
     assoc(key, options[key] ? selected : exists, tests)
 
-  const req = id ? required :
-    new_required.concat(
+  const req = (id ? required : new_required)
+    .concat(
```

Comment:
Shouldn't it be autogenerated as today's date, so the field doesn't need to be required?